src/components/FormStepSummary/ComponentValueDisplay.js

```diff
@@ -219,6 +219,10 @@ const AddressNLDisplay = ({component, value}) => {
   }`;
 };
 
+const ContentDisplay = ({component, value}) => {
+  return <span dangerouslySetInnerHTML={{__html: component.html}} />;
```

Comment:
or `__html: value`, but this would require a change in the backend: `Submission.render_summary_page` currently uses `Renderer(..., as_html=False)`, hence `value` is just the string value without html and styling

Reply:
I think it makes sense to use `component.html` :thinking:

Reply:
One thing that I'm not sure about is if that HTML is CSP post-processed to convert inline styles (like text color) to CSP-nonced `<style>` tags. I can't find any reference in the `submissions` code that would hint this is the case.

I'm okay with leaving this out, as most of the formatting will still work since they're applied through `<strong>` tags etc. rather than inline styles.

Reply:
It's a bit weird, the summary endpoint returns the `html` without `style` tags, e.g.:
`<p>Some <i>pretty </i><span style="color:#4c99e6;"><i>component</i></span></p>`

But the submission step endpoint uses `style` elements:

`<style nonce="RqgbALvp8D5b3+8NuhfuKg==">#nonce-a2ef23e2ee96a66e5e16b1c2c7aec411-139719785812288 {    color:#4c99e6;} </style><p>Some <i>pretty </i><span id="nonce-a2ef23e2ee96a66e5e16b1c2c7aec411-139719785812288"><i>component</i></span></p>`